### PR TITLE
[Keymap] VIA Support for Nein Macropad

### PR DESCRIPTION
**Pull request by thompson-ele**
## Description
Added VIA support for the Nein 3x3 macropad PCB. Also updated the vendor ID from 0xFEED to match the vendor ID's declared in other 40% club keyboard config.h files.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/40percentclub/nein/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ),
+    [2] = LAYOUT_ortho_3x3(
+        _______, _______, _______, \
+        _______, _______, _______, \
+        _______, _______, _______ \
+    ),
+    [3] = LAYOUT_ortho_3x3(
+        _______, _______, _______, \
+        _______, _______, _______, \
+        _______, _______, _______ \
```
Comment (fauxpark) [inline keyboards/40percentclub/nein/keymaps/via/keymap.c:37]:
```suggestion
        KC_MUTE, KC_HOME, KC_MPLY,
        MO(1),   KC_UP,   KC_END,
        KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT_ortho_3x3(
        RESET,   _______, KC_STOP,
        _______, _______, RGB_MOD,
        KC_MPRV, _______, KC_MNXT
    ),
    [2] = LAYOUT_ortho_3x3(
        _______, _______, _______,
        _______, _______, _______,
        _______, _______, _______
    ),
    [3] = LAYOUT_ortho_3x3(
        _______, _______, _______,
        _______, _______, _______,
        _______, _______, _______
```

File: keyboards/40percentclub/nein/keymaps/via/rule.mk [hunk 2]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/40percentclub/nein/keymaps/via/rule.mk:1]: Should be named `rules.mk`

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/40percentclub/nein/keymaps/via/rule.mk (hunk 2 above)
Comment (thompson-ele) [reply keyboards/40percentclub/nein/keymaps/via/rule.mk]: @fauxpark omg i'm special lol I've updated the file name. thanks!

**Review by fauxpark — APPROVED** [review]

**Review by Erovia — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**